src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -140,6 +145,183 @@ class_implements($class_name)
   /**
    * {@inheritdoc}
    */
+  public function value(DataSourceInterface $source) {
+    // Return if this field is a callback.
+    $value = $this->decorated->value($source);
+    if (isset($value)) {
+      return $value;
+    }
+
+    if ($resource = $this->getResource()) {
+      // TODO: The resource input data in the field definition has changed.
+      // Now it does not need to be keyed by bundle since you don't even need
+      // an entity to use the resource based field.
+
+      $request = Request::create('', array(), Request::METHOD_GET);
+      $resource_data_provider = DataProviderResource::init($request, $resource['name'], array(
+        $resource['majorVersion'],
+        $resource['minorVersion'],
+      ));
+      // FIXME: $embedded_identifier needs to be fetched first!!!
+      return $resource_data_provider->view($embedded_identifier);
```

Comment:
To se if I understand correctly - embedded_identifier should be the ID for the sub-resource? e.g. ``$resource->getIdentifier``?